src/style.css

```diff
@@ -1,3 +1,75 @@
+/* stylelint-disable no-descending-specificity */
+/* stylelint-disable color-hex-case */
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+[class*="-container"] {
+  max-width: 1200px;
```

Comment:
it's generally a bad idea to hardcode width, if user has a wide screen than your header would have massive indents, and it would look weird
<img width="1033" alt="Знімок екрана 2023-07-14 о 14 43 06" src="https://github.com/mate-academy/layout_moyo-header/assets/106310796/6b2e3f75-c712-4dab-b80e-858de9f30b77">